src/main/java/net/minestom/server/particle/ParticleOptionImpl.java

```diff
@@ -0,0 +1,392 @@
+package net.minestom.server.particle;
+
+
+import net.minestom.server.color.Color;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import static net.minestom.server.particle.ParticleOption.*;
+
+public interface ParticleOptionImpl {
```

Comment:
Why is this called ParticleOptionImpl?
Generally, classes ending in Impl shouldn't be exposed publicly at all.
Perhaps `ParticleOptions` would be more fitting?

Reply:
I think so too. Renamed